opflexagent/gbp_ovs_agent.py

```diff
@@ -112,6 +123,20 @@ def __init__(self, root_helper=None, **kwargs):
         # TODO(ivar): make this component pluggable.
         self.ep_manager = ep_manager.EndpointFileManager().initialize(
             self.host, self.bridge_manager, kwargs)
+        self.init_firewall()
+
+    def _invoke_ep_mgr(self, port):
```

Comment:
what about loading a void EP manager when VCenter is used? this would reduce by a decent amount the code change in this class, which is the most overloaded one, by adding an EP manager that simply inherits from the base and does basically nothing.